app/models/direct_file_json_data.rb

```diff
@@ -36,6 +36,10 @@ def initialize(json)
     @data = JSON.parse(json || "{}")
   end
 
+  def to_s
+    @data.to_s
+  end
```

Comment:
why was this added?

Reply:
Good catch. I originally added it because I incorrectly used it on `state_file_id_intakes.rb:122` instead of `to_json`, but now that I look more closely, I introduced a bug when I switched over to calling `to_json`, as it's now returning a json string of the entire `DirectFileJsonData` object (i.e. it has `{data: {...}}`) instead of just its stored json object. So this needs to be changed to a `to_json` that exposes `@data`, e.g.

```
def to_json
  @data.to_json
end
```

or
```
delegate :to_json, to: :data
```

Reply:
@mrotondo does this still need to be addressed?

Reply:
done & added a test to verify